tomviz/operators/ArrayWranglerOperator.cxx

```diff
@@ -135,11 +135,14 @@ void wrangleVtkArrayTypeUnsigned(vtkOutputArrayType* array, int nComps,
 }
 
 template <typename vtkOutputArrayType>
-void applyGenericWrangleTransform(vtkImageData* imageData, int componentToKeep,
-                                  double range[2])
+void applyGenericWrangleTransform(vtkImageData* imageData, int componentToKeep)
```

Comment:
This is already in an anonymous namespace, don't repeat wrangle too much, it is already in the file name, and applyGenericTransform is already quite obvious.